os/board/stm32l4r9ai-disco/src/Makefile

```diff
@@ -89,6 +89,13 @@ VPATH += :$(TOPDIR)/board/common
 CSRCS += partitions.c
 endif
 
+DEPPATH += --dep-path $(TOPDIR)/board/common
+VPATH += :$(TOPDIR)/board/common
+
+ifeq ($(CONFIG_BOARD_CRASHDUMP),y)
+CSRCS += stm32_crashdump.c
```

Comment:
1. What is the difference between crashdump.c and stm32_crashdump.c? Could you let me know why it is necessary newly?
2. If this is necessary, you should not add it in common. You should put it in stm folder becaues it is not common.

Reply:
The low level API's used in stm32_crashdump.c are implemented in hal layer (stm32l4xx_hal_interface.c ), there are separate up_lowptc() & up_getc() implementations in stm32l4_serial.c
If we wish to override these implementations in serial file with the ones in stm32l4xx_hal_interface.c , then we can move it to common file crashdump.c else we will have to keep it separate.
What is your opinion ?

Reply:
File name is not important. Anyway stm also provides up_lowputc and up_getc in arch as others do. Then there is no problem to use common. It looks the same as common.

Reply:
Ramdump fails with the up_lowptc() & up_getc() implementations in stm32l4_serial.c.
So shall I go ahead and modify them ?